mdx-models/src/main/java/com/mx/path/model/mdx/model/account/Features.java

```diff
@@ -14,9 +14,9 @@ public class Features extends MdxBase<Features> {
   private Boolean hasEditAddress;
   private Boolean hasEditNickname;
   private Boolean hasFeature;
+  private Boolean hasManageStatementsDeliveryPreference;
   private Boolean hasOverdraft;
   private Boolean hasOverdraftEnrollment;
   private Boolean hasStatements;
-  private Boolean hasStatementsEnrollment;
```

Comment:
As this is a breaking change technically, can you comment here and ensure that it's not a field in use by any service, except the ones you're working on?

Reply:
I have received confirmation that this isn't a breaking change.